hcfb-be/backend-server/app/controller/distribution.controller.js

```diff
@@ -0,0 +1,75 @@
+const db = require('../config/db.config.js');
+const Distribution = db.distributions;
+
+exports.create = (req, res) => {
+    //Save to postgre database
+    Distribution.create({
+      "deliveryLocation" : req.body.deliveryLocation,
+      "deliveryDepartureTime" : req.body.deliveryDepartureTime,
+      "deliveryWeights" : req.body.deliveryWeights,
+      "projectedNumberOfClients" : req.body.projectedNumberOfClients
+    }).then(distribution => {
+        res.json(distribution);
+    }).catch(err => {
+        console.log(err);
+        res.status(500).json({msg: "error", details: err});
+    });
+};
+
+// Find a the most recent distribution
+exports.findByLocation = (req, res) => {	
+    Distribution.findAll(
+        { where: {deliveryLocation: req.params.deliveryLocation} }).then(distribution => {
+			res.json(distribution);
+		}).catch(err => {
+			console.log(err);
+			res.status(500).json({msg: "error", details: err});
+		});
+};
+
+// Fetch all Distributions
+exports.findAll = (req, res) => {
+	Distribution.findAll().then(distributions => {
+			// Send All Distributions to Client
+			res.json(distributions.sort(function(c1, c2){return c1.id - c2.id}));
```

Comment:
```suggestion
			res.json(distributions.sort((c1, c2) => c1.id - c2.id));
```

Just a tad terser. (And a little less surprising if you end up needing to use `this` in here for some reason later.)